specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2516,6 +2674,144 @@
+      "description": "Error response indicates CDN service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "errors":{
+          "items": {
+            "$ref": "#/definitions/MigrationErrorType"
+          },
+          "type": "array"
+        }
+      }
+    },
+    "MigrationErrorType": {
+      "description": "Error response indicates CDN service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "resourceName": {
+          "description": "Resource which has the problem.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "errorMessage": {
+          "description": "Error message indicating why the operation failed.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "nextSteps": {
+          "description": "Describes what needs to be done to fix the problem",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "MigrationParameters": {
+      "type": "object",
+      "description": "Request body for Migrate operation.",
+      "properties": {
+        "Sku": {
+          "description": "Sku for the migration",
+          "type": "string",
+          "enum": [
+            "Standard_AzureFrontDoor",
+            "Premium_AzureFrontDoor"
+          ],
+          "x-ms-enum": {
+            "name": "MigrationSku",
+            "modelAsString": true
+          }
+        },
+        "ProfileName": {
+          "description": "Name of the new profile that need to be created.",
+          "type": "string"
+        },
+        "MigrationWafMappings": {
+          "description": "Waf mapping for the migrated profile",
+          "type": "array",
+          "items": {
+            "description": "List of waf mappings",
+            "$ref": "#/definitions/MigrationWafMapping"
+          }
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "MigrationWafMapping": {
+      "description": "Waf mapping",
+      "type":"object",
+      "properties": {
+        "MigratedFrom": {
+          "description": "Migration From Waf policy",
+          "$ref": "#/definitions/ResourceReference"
+        },
+        "MigratedTo": {
+          "description": "Migration to Waf policy",
+          "$ref": "#/definitions/ResourceReference"
+        }
+      }
+    },
+    "MigrateResult": {
+      "type": "object",
+      "description": "Result for canMigrate operation.",
+      "properties": {
+        "MigratedProfileArmResourceId": {
+          "readOnly": true,
+          "description": "Arm resource id of the migrated profile",
+          "$ref": "#/definitions/ResourceReference"
+        },
+        "MigratedWafArmResourceIds": {
```

Comment:
please remove this MigratedWafArmResourceIds for MigrationResult per our discussion.